content/docs/5.1/concepts/explore-tikv-features/replication-and-rebalancing.md

```diff
@@ -141,19 +151,19 @@ In this section, we will launch a larger workload, then scales the 3-node local
 
 ## Step 6: Verify the data rebalance
 
-Go to Grafana page mentioned above. You will find some regions are split and rebalance to the two new nodes.
+Go to the Grafana page as mentioned above. You can find some Regions are split and rebalanced to the two new nodes.
 
 {{< figure
     src="/img/docs/data-rebalance.png"
     width="80"
     number="1" >}}
 
+## Step 7: Stop and delete the cluster
 
```

Comment:
For the two steps in this section, they are almost the same as that in "https://github.com/tikv/website/pull/250". Please refer to "https://github.com/tikv/website/pull/250" to update them so we can keep similar content consistent.

Reply:
Hmmm, I don't think so. It might be a little verbose to keep the page self-explained.